docs/setting_up_dev_env.rst

```diff
@@ -0,0 +1,18 @@
+This document describes steps for getting the Plant Tracer webapp development up and running on MacOS
+# Getting Started
+
+## Read the Docs
+Read and be familiar with the documentation for venv:
+
+- [ ] [venv](https://docs.python.org/3/library/venv.html), the Python virtual environment manager used for this project
+
+
+## Run venv Locally
+The following commands will help you get the virtual development environment up and running locally:
+
+- [ ] `make install-macos`
+- [ ] `python -m venv venv`
+
+This should get you to a good starting point for developing for Plant Tracer
```

Comment:
This shouldn't be necessary.  This is the `make venv` target in the Makefile

Reply:
Gotcha, I missed that in the makefile.